src/blockchain/shelley/certificate.js

```diff
@@ -28,9 +28,9 @@ export type PoolRegistrationType = {
   operators: Array<string>,
   rewardAccount: string|null,
   rewards: {
-    fixed: number,
-    ratio: [number, number],
-    limit: number|null,
```

Comment:
I think pool taxation explicitly **must** contain the fixed part and the ration, only limit is optional